contracts/interfaces/IClearingTypes.sol

```diff
@@ -6,9 +6,9 @@ import '@statechannels/nitro-protocol/contracts/interfaces/INitroTypes.sol';
 /**
  * @notice Interface
  */
-interface IMarginApp {
+interface IClearingTypes {
```

Comment:
Maybe ISettlement ?

Reply:
and IMargin so we separate the two types

Reply:
17fa384